crates/sui-kvstore/src/bigtable/client.rs

```diff
@@ -462,52 +488,116 @@ impl BigTableClient {
     ) -> Result<Vec<Vec<(Bytes, Bytes)>>> {
         let start_time = Instant::now();
         let num_keys_requested = keys.len();
-        let result = self.multi_get_internal(table_name, keys).await;
+        let result = self.multi_get_internal(table_name, keys, None).await;
```

Comment:
Can we avoid duplicating the metrics code? 
A single `multi_get` method with an optional filter should be enough. There are just a few callsites, so it shouldn't be an inconvenience